portfolio/src/main/webapp/index.html

```diff
@@ -43,6 +43,18 @@ <h2 class="greeting">Hi, I'm Karol Gutiérrez</h2>
 
       <!--COMMENTS-->
       <h2>Comments</h2>
+
+      <label for="comments-language">Choose a language:</label>
```

Comment:
I was not aware of this for attribute before, thanks!

Should this be "languages"?  According to doc, this must be the related element.
https://www.w3schools.com/tags/tag_label.asp